src/main/java/net/mcreator/ui/validation/AggregatedValidationResult.java

```diff
@@ -41,55 +41,29 @@ public AggregatedValidationResult(Collection<ValidationGroup> validationGroups)
 		this.validationGroups.addAll(validationGroups);
 	}
 
-	@Override public <T extends JComponent & IValidable> void addValidationElement(T validable) {
+	@Override public <T extends JComponent & IValidable> ValidationGroup addValidationElement(T validable) {
 		validationElements.add(validable);
+		return this;
 	}
 
-	public void addValidationGroup(ValidationGroup validable) {

```

Comment:
This method is still used by #4400:
https://github.com/MCreator/MCreator/blob/2e27bebcb536bdfb6099ffa67ee6a2e82cbacf61/src/main/java/net/mcreator/ui/minecraft/jigsaw/JJigsawPoolsList.java#L88-L90
Should it be refactored to something like this?
```java
		return new AggregatedValidationResult(entryList.stream().map(JJigsawPool::getValidationResult)
				.toArray(AggregatedValidationResult[]::new));
```

Reply:
![image](https://github.com/MCreator/MCreator/assets/16374228/067c126e-da93-4fcf-bc15-f5b11768b951)

JModelLayerList co-authored by you uses this approach so I would prefer to keep things consistent across the codebase

Reply:
`JModelLayerList` is a simple list unlike `JLootTablePoolsList`, `JVillagerTradeProfessionsList` or, in this case, `JJigsawPoolsList`. It's just that neither loot table entries nor trade entries needed such validation, whereas jigsaw parts did.

Reply:
I don't see how different complexity prevents using approach mentioned above